src/profile/components/editingNotifications/EditingNotifications.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import Notification from '../../../common/copyOfHDSNotification/Notification';
+import { NotificationContent } from './useNotificationContent';
+import styles from './EditingNotifications.module.css';
+import { EditDataType } from '../../helpers/editData';
+
+type Props = {
+  content: NotificationContent;
+  dataType: EditDataType;
+};
+
+function EditingNotifications({
+  content,
+  dataType,
+}: Props): React.ReactElement | null {
+  if (!content.text) {
```

Comment:
According to clean code passing or returning null is bad: https://enos.itcollege.ee/~jpoial/oop/naited/Clean%20Code.pdf#page=141

Because then you have to do these checks inside components and they don't render what you would expect them to render. Maybe one option is to check the content.text value in BasicData.tsx, and only render this EditingNotifications if it has a value 🤔

Reply:
I think it is normal in React world to render nothing (=return null), if component don't have correct data etc. to render its content. Developer can see it in return type definition.

This is a common component which will be used by also by address, phones, emails editors. It is easier and clearer to check content here than every time component is used. 

BasicData:
{ if !context.text && <EditingNotifications .../> }

Addresses:
{ if !context.text && <EditingNotifications .../> }

and so on...

And if there is a new prop (content.alert) that must be added to each implementation instead of in the component itself.

{ if !context.text && !content.alert && <EditingNotifications .../> }